xml/ha_storage_protection.xml

```diff
@@ -1373,6 +1373,271 @@ Illegal request, Invalid opcode</screen>
   </sect2>
  </sect1>
 
+ <sect1 xml:id="sec-ha-storage-protect-sbd-redeploy">
+  <title>Changing SBD configuration</title>
+  <para>
+    You might need to change the cluster's SBD configuration for various reasons. For example:
+  </para>
+  <itemizedlist>
+   <listitem>
+     <para>
+       Changing disk-based SBD to diskless SBD
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Changing diskless SBD to disk-based SBD
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Replacing an SBD device with a new device
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Changing timeout values and other settings
+     </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+    You can use &crmsh; to change the SBD configuration. However, this method uses &crmsh;'s
+    default settings, including timeout values. If you need to change any settings, or if you use
+    custom settings and need to retain them when replacing a device, you must manually edit the
+    SBD configuration. Both methods are described in the following procedures.
+  </para>
+  <procedure xml:id="pro-ha-storage-protect-sbd-redeploy-disk-based">
+    <title>Redeploying disk-based SBD</title>
+    <para>
+      Use this procedure to change diskless SBD to disk-based SBD, to replace an existing
+      SBD device with a new device, or to change the timeout settings for disk-based SBD.
+    </para>
+    <step>
+      <para>
+        Put the cluster into maintenance mode:
+      </para>
+<screen>&prompt.root;<command>crm configure property maintenance-mode=true</command></screen>
```

Comment:
If it is really needed, we can use the simpler interface

`crm maintenance on/off`

Reply:
Thanks @zzhou1! Which versions can use this simpler method? Maintenance mode is mentioned in a few other places, so when this bug is done I'll open another doc task to change the command everywhere else too.

Reply:
Sure. It long exists since SLE12.

Reply:
Thanks, I'll backport the simpler command all the way to 12 SP5 then :)